lib/foreman_maintain/cli/report_command.rb

```diff
@@ -70,10 +70,10 @@ def aggregate_host_count(data)
           rhel_count = data['hosts_by_os_count|RedHat'] || 0
           rh_count = data['hosts_by_family_count|Redhat'] || 0
           result['host_rhel_count'] = rhel_count
-          result['host_redhat_count'] = rh_count - rhel_count
+          result['host_redhat_count_without_rhel'] = rh_count - rhel_count
```

Comment:
I'd like to keep the `_count` suffix, would this do?
```suggestion
          result['host_redhat_without_rhel_count'] = rh_count - rhel_count
```